utils/sendpart/sendpart.go

```diff
@@ -37,16 +38,15 @@ type DataMultipartWriter interface {
 	httputil.MultipartWriter
 }
 
-// POST sends a POST request using client to the given URL and unmarshal the
-// body into v if it's not nil. It will only send using multipart if files is
-// true.
-func POST(c *httputil.Client, data DataMultipartWriter, v interface{}, url string) error {
+// Do sends an HTTP request using client to the given URL and unmarshals the
+// body into v if it's not nil. It will only send using multipart if needed.
+func Do(method string, c *httputil.Client, data DataMultipartWriter, v interface{}, url string) error {
```

Comment:
The order of these arguments should be `Do(c *httputil.Client, method string, data DataMultipartWriter, v interface{}, url string) error`.